cargo-pgrx/src/command/test.rs

```diff
@@ -140,6 +150,14 @@ pub fn test_extension(
         .env("PGRX_BUILD_PROFILE", profile.name())
         .env("PGRX_NO_SCHEMA", if no_schema { "true" } else { "false" });
 
+    if let Some(runas) = runas {
+        command.env("CARGO_PGRX_TEST_RUNAS", runas);
```

Comment:
Oh environment variable support is handy

Reply:
Yeah, you could turn this dial in the case where you're not using cargo-pgrx.  Say if you wanted to test a complex rust workspace crate that happens to include one or more pgrx extensions too:

```shell
$ CARGO_PGRX_TEST_PGDATA=/tmp/pgrx-pgdata \
   CARGO_PGRX_TEST_RUNAS=theory \
   cargo test --all --features pg14 --no-default-features
```